data/mships.txt

```diff
@@ -410,8 +410,8 @@ ship "CY-10 Dragonfly" #Base variant without carry.
 		under
 	explode "tiny explosion" 10
 	explode "small explosion" 15
-	description "Small and agile heavy-lift spacecraft. Can be used for transporting containers or as method of quickly delivering or recovering fighters. Also often used for lifting within atmosphere on various occasion including  assisting construction of tall buildings or delivering supplies to inaccessible area."
-	description `[Carry: Nothing, you shouldnt see or use this]`
+	description "This is a small and agile heavy-lift spacecraft. It can be used for transporting containers or as a method of quickly delivering or recovering fighters. Also, it is often used for lifting within atmosphere on various occasions, including assisting in the construction of tall buildings or delivering supplies to inaccessible areas."
+	description `	[Carries: nothing, you shouldn't see or use this.]`
```

Comment:
```suggestion
	description "This is a small and agile heavy-lift spacecraft. It can be used for transporting containers or as a method of quickly delivering or recovering fighters. It is also often used for lifting within atmosphere on various occasions, including assisting in the construction of tall buildings or delivering supplies to inaccessible areas."
	description `	[Carries: Nothing, you shouldn't see or use this.]`
```